python/ngen_conf/src/ngen/config/init_config/cfe.py

```diff
@@ -0,0 +1,165 @@
+from typing import TYPE_CHECKING, List, Literal, Optional, Union
+
+from ngen.init_config import serializer_deserializer as serde
+from ngen.init_config import validators
+from pydantic import Field, validator
+
+from .utils import FloatUnitPair
+
+if TYPE_CHECKING:
+    from pydantic.typing import AbstractSetIntStr, DictStrAny, MappingIntStrAny
+
+
+m = Literal["m"]
+m_per_m = Literal["m/m"]
+empty = Literal[""]
+
+
+class CFEBase(serde.IniSerializerDeserializer):
+    forcing_file: Literal["BMI"] = "BMI"
+    # soil depth
+    soil_params_depth: FloatUnitPair[m]
+    # beta exponent on Clapp-Hornberger (1978) soil water relations
+    soil_params_b: FloatUnitPair[empty]
+    # saturated hydraulic conductivity
+    soil_params_satdk: FloatUnitPair[Literal["m s-1"]]
+    # saturated capillary head
+    soil_params_satpsi: FloatUnitPair[m]
+    # this factor (0-1) modifies the gradient of the hydraulic head at the soil bottom. 0=no-flow.
+    soil_params_slop: FloatUnitPair[m_per_m]
+    # saturated soil moisture content
+    soil_params_smcmax: FloatUnitPair[m_per_m]
+    # wilting point soil moisture content
+    soil_params_wltsmc: FloatUnitPair[m_per_m]
```

Comment:
the parameter `refkdt` is missing here (optional parameter), the default value is 3.0 set inside cfe.c